src/core/templates/image.jsx

```diff
@@ -0,0 +1,40 @@
+import Adapt from 'core/js/adapt';
+import React from 'react';
+import { html, classes, prefixClasses } from 'core/js/reactHelpers';
+
+/**
+ * Size switching content image
+ * @param {Object} props
+ * @param {Array} props.classNamePrefixes
+ * @param {Array} [props.attributionClassNamePrefixes]
+ */
+export default function Image(props) {
+  const screenSize = Adapt.device.screenSize;
+  const attributionClassNamePrefixes = (props.attributionClassNamePrefixes || props.classNamePrefixes);
+  const isSet = (props[`_${screenSize}`] || props[`${screenSize}`] || props._src || props.src);
+  if (!isSet) return null;
+  return (
+    <div className={classes([
+      prefixClasses(props.classNamePrefixes, ['-image-container']),
+      props.attribution && 'has-attribution'
+    ])}>
+
+      <img
+        className={prefixClasses(props.classNamePrefixes, ['-image'])}
+        src={props[`_${screenSize}`] || props[`${screenSize}`] || props._src || props.src}
```

Comment:
Minor: Could you tweak the isSet check above to reduce the repetition of this logic?